app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def update_posts_counter
+    post.increment!(:PostsCounter)
```

Comment:
- [ ] Hello @Chimwemwe-127001 nice work ut you did a typo error here at line 7 for the increment, you should use the same as your database. `:Posts_Counter`

Reply:
Thanks for spotting that

Reply:
Done